src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -235,15 +241,15 @@ <h2>
                             Online editor
                         </button>
                         <button
-                            *ngIf="exercise.course && isManualResult(value)"
+                            *ngIf="exercise.course"
                             [routerLink]="['/course-management', exercise.course?.id, 'programming-exercises', exercise.id, 'code-editor', value.participation.id, 'assessment']"
                             class="btn btn-primary btn-sm mr-1"
                         >
                             <fa-icon [icon]="'folder-open'" [fixedWidth]="true"></fa-icon>
                             {{ 'artemisApp.programmingSubmission.view' | translate }}
                         </button>
                         <button
-                            *ngIf="exercise.exerciseGroup && isManualResult(value)"
+                            *ngIf="exercise.exerciseGroup"
```

Comment:
We also need to disable that for purely automatic programming exercise.

Reply:
Fixed in `70289fe`